webpage/index.html

```diff
@@ -802,19 +790,17 @@ <h2>MQTT</h2>
             <div class="pure-u-24-24 section section-about">
                 <div class="header">
                     <svg><use href="#info"/></svg>
-                    <h1>Über</h1>
+                    <h1 data-i18next="about.h1"></h1>
                 </div>
 
                 <div class="box">
-                    <p>Die Software dieser Wortuhr basiert auf der Wortuhr von <a href="https://web.archive.org/web/20180422160812/http://www.ulrichradig.de/home/index.php/projekte/wort-uhr-neu">Ulrich Radig (Stand 2019)</a> und wird nun weiterentwickelt auf <a href="https://github.com/ESPWortuhr/Wortuhr">GitHub</a> von <a
-                            href="https://github.com/Eisbaeeer">Lars Weimar</a>, <a href="https://github.com/Wandmalfarbe">Pascal Wagler</a>, <a href="https://github.com/dbambus">dbambus</a>, <a href="https://github.com/shortN0te">shortN0te</a>,
-                        <a href="https://github.com/Flo455">Flo455</a>, <a href="https://github.com/Elektron79">Elektron79</a>, <a href="https://github.com/tali">tali</a>, <a href="https://github.com/atho95">ATho95</a>  und <a href="https://github.com/masju1">masju1</a>.</p>
+                    <p data-i18next="about.text"></p>
                         <p>Ein Update der Software ist möglich, dazu bitte <a href="/update" onmouseover="javascript:event.target.port=81">auf die Update-Seite wechseln</a>.</p>
```

Comment:
I would like to move the update functionally to the settings page because the about page seems like the wrong one. I'll provide the translation after that change in a separate PR.